### DISCUSSION TITLE
For which features is the UberViz module actually needed?

### DISCUSSION THREAD

**Discussion by DJ-SteffEN** [ampache/ampache#3936] [Q&A]
Hello,

the JS functions 'SlideShow', 'Visualizer' and Co have not been working for me for a long time ([inthemix.at](https://inthemix.at)). They don't work in the current demo version either, perhaps this is intentional. I removed the corresponding controls in the player some time ago.

Now I wonder if the UberViz module is still needed at all. Can I safely remove the JS files or are they still needed for other features?

Thanks for your help
DJ Steffen

**Comment by lachlan-00** [comment]
i see its not working on demo but i've checked on develop and patch7 right now and it's working okay.

I had to update the CSS to get it on top but it's functioning there again

![image](https://github.com/ampache/ampache/assets/1305249/f8e290fb-2216-4160-9a82-4281dc0af100)

**Comment by lachlan-00** [comment]
slideshow doesn't work, i forget what that is, just hte album art?

**Reply by mitchray** [reply]
I think if the API key is set up, it will show images from Flickr
